google/ggrc-core
# GGRC-6961 Remove usage of 'can.view.hook'

# Issue description

'can.view' methods are deprecated in CanJs v3 and should be replaced with another supported functionality.

# Steps to test the changes

Case#1:
- Open Dashboard page & expand LHN menu
- Expand 'Issues' section & hover over any item

Expected behavior: 'Extended Info' modal contains info about mapped items

Case#2:
- Open Dashboard page
- Click 'Create a Task' button
- Fill out all mandatory fields & click 'Save' button

Expected behavior: New Task is successfully created.

# Solution description

Replace usage of 'with_mapping_count' helper w/ appropriate component ('mapped-counter')
Replace usage of 'using' helper w/ newly created <object-loader> component that explicitly fetches data on demand.

# Sanity checklist




# PR Review checklist
